coordinator/src/routes/prescription/post-prepare-message.ts

```diff
@@ -13,9 +14,11 @@ export default [
     method: "POST",
     path: "/$prepare",
     handler: validatingHandler(
-      (requestPayload: Bundle, _: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) => {
+      (requestPayload: Bundle, request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit) => {
+        const isSmokeTest = request.headers["x-smoke-test"]
+        const contentType = isSmokeTest ? CONTENT_TYPE_JSON : CONTENT_TYPE_FHIR
```

Comment:
Why did we need to add this? I'm sure there's a good reason, just curious

Reply:
In order to use the same pact verifier for all pact tests had to add a workaround for the content-type detection, as the latest pact verififer was unable to work out the content-type "application/fhir+json" so the diffs weren't working and pact would fail body matching for everything. Hopefully they'll fix in future version

Reply:
Ah fair enough